srcs/juloo.keyboard2/KeyValue.java

```diff
@@ -418,6 +418,9 @@ public static KeyValue getKeyByName(String name)
       case "ole_placeholder": return placeholderKey(Placeholder.OLE);
       case "meteg": return charKey("\u05DE\u05BD", '\u05BD', 0); // or siluq or sof-pasuq
       case "meteg_placeholder": return placeholderKey(Placeholder.METEG);
+      /* intending/preventing ligature - supported by many scripts*/
+      case "zwj": return charKey("zwj", '\u200D', 0); // zero-width joiner (provides ligature)
+      case "zwnj": return charKey("zwnj", '\u200C', 0); // zero-width non joiner (prevents unintended ligature)
```

Comment:
`zwj` and `zwnj` could have nicer symbols. They are added to `srcs/special_font` as SVG files and rebuilt with `rebuild_special_font`, which requires `fontforge` installed. The char code is specified by the file name, see how `modifierKey` has a version that takes a `int symbol` instead of a `String symbol`.

I can take care of this later if you don't have the time.

Reply:
If you want to replace with symbols, you can. But I have seen layouts that have these characters, labeled with zwj and zwnj.